exercises/accumulate/src/lib.rs

```diff
@@ -1,4 +1,9 @@
+// Note: The tests for this exercise are not #[ignore]d,
+//       but commented out with a multi-line comment
+         /* this is a multi-line style comment */
+//       To activate a test, move the comment starter "/*" below it.
+
 /// What should the type of _function be?
-pub fn map(input: Vec<i32>, _function: ???) -> Vec<i32> {
+pub fn map(input: Vec<i32>, function: ???) -> Vec<i32> {
```

Comment:
Eliminating the leading underscore from `_function` will cause the compiler to complain about an unused variable. Despite the fact that this exercise is allowed to not compile, I'd prefer to minimize the number of compiler errors the student gets when they first download the exercise.

Reply:
I'll add it back, although it barely changes anything what with all the other compiler errors.